string input_str
---
string output_str